documentation/asciidoc/accessories/camera/libcamera_options_common.adoc

```diff
@@ -145,6 +162,25 @@ Using this option often incurs a frame rate penalty, as larger resolution frames
 
 Example: `libcamera-raw -t 2000 --segment 1 --rawfull -o test%03d.raw` will cause multiple full resolution raw frames to be captured. On the HQ camera each frame will be about 18MB in size. Without the `--rawfull` option the default video output resolution would have caused the 2x2 binned mode to be selected, resulting in 4.5MB raw frames.
 
+----
+	--mode				Specify sensor mode, given as <width>:<height>:<bit-depth>:<packing>
+----
+
+This option is more general than `--rawfull` and allows the precise selection of one of the camera modes. The mode should be specified by giving its width, height, bit-depth and packing, separated by colons. These numbers do not have to be exact as the system will select the closest it can find. Moreover, the bit-depth and packing are optional (defaulting to 12 and `P` for "packed" respectively). For example:
+
+* `4056:3040:12:P` - 4056x3040 resolution, 12 bits per pixel, packed. This means that raw image buffers will be packed so that 2 pixel values occupy only 3 bytes.
+* `1632:1224:10` - 1632x1224 resolution, 10 bits per pixel. It will default to "packed". A 10 bit packed mode would store 4 pixels in every 5 bytes.
+* `2592:1944:10:U` - 2592x1944 resolution, 10 bits per pixel, unpacked. An unpacked format will store every pixel in 2 bytes, in this case with the top 6 bits of each value being zero.
+* `3264:2448` - 3264x2448 resolution. It will try to select the default 12 bit per mode but in the case of the v2 camera there isn't one, so a 10 bit mode would be chosen instead.
```

Comment:
"12 bit per mode" -> "12 bit mode"?  Also, should "12 bit" be hyphenated as 12-bit ?

Reply:
Indeed. I think I'll go with "<n> bits" when "bits" is obviously a noun, but "<n>-bit" where it's combined as more of an adjective. So "12 bits per pixel" but "12-bit pixel". Does that seem plausible?